watertap/data/techno_economic/tests/test_unit_parameter_files.py

```diff
@@ -102,6 +103,8 @@ def test_unit_parameter_files(tech):
         "water_pumping_station",
         "well_field",
         "mbr",
+        "vfa_recovery",
+        "cofermentation",
```

Comment:
It would be preferable to maintain the alphabetical order here, so it's easy to know if something is in the list by inspection.